cmake/iwyu.cmake

```diff
@@ -29,10 +29,10 @@ if(NOT WIN32)
       COMMAND "${Python3_EXECUTABLE}" "${iwyu_tool_py}" -p iwyu_commands.json -o iwyu
         -j ${cpu_count} -- -w
         -Xiwyu "--mapping_file=${CMAKE_SOURCE_DIR}/contrib/iwyu/orbit.imp"
-        -Xiwyu --cxx17ns -Xiwyu --max_line_length=120 -Xiwyu --no_fwd_decls > include-what-you-use.log
+        -Xiwyu --cxx17ns -Xiwyu --max_line_length=120 -Xiwyu --no_fwd_decls | tee include-what-you-use.log
```

Comment:
We initially introduced that for debug purposes. Do you want to keep it or is this just an oversight? (both fine with me)

Reply:
I found it super useful to be honest. So I would keep it.